openmetadata-ui/src/main/resources/ui/src/components/EntityLineage/CustomNode.component.tsx

```diff
@@ -47,7 +47,7 @@ const getHandle = (
           id={id}
           isConnectable={isConnectable}
           position={Position.Left}
-          style={handleStyles}
+          style={{ ...handleStyles, borderLeft: '5px solid #d9ceee', left: 0 }}
```

Comment:
Create a separate style object for the left and right border handles and use it here and in other places.